src/vs/workbench/contrib/positronNotebook/browser/positronNotebook.contribution.ts

```diff
@@ -435,10 +435,9 @@ registerCellCommand({
 	commandId: 'positronNotebook.cell.runAllAbove',
 	handler: (cell, notebook) => {
 		const cells = notebook.cells.get();
-		const cellIndex = cells.indexOf(cell);
 
 		// Run all code cells above the current cell
-		for (let i = 0; i < cellIndex; i++) {
+		for (let i = 0; i < cell.index; i++) {
```

Comment:
My only concern here happens due to the fact that this is not a reference instead of a direct value and thus _could_ change mid-loop. I'm not sure if that would cause issues or in fact is a benefit here. Just worth thinking about.